### PR TITLE
Document the applicant review process

### LINKED ISSUE #83: Clarify the wording we should use for onboarding calls
We want to make joining hackEDU feel "exclusive" and introduce scarcity when talking with people.

### LINKED ISSUE #312: Document the applicant review process

### PR DESCRIPTION
**Pull request by zachlatta**
This pull request better documents our applicant review process. It's not great, but this will be a working document that we'll improve as we work with more people.

This closes #312 and also closes #83 (it doesn't do a great job of clarifying the wording to use in the call, but at least it says what the call's about and what the interviewer should be looking for).

### CODE REVIEW

**Comment by jonleung** [comment]
@zachlatta, what's the best way to get these spelling and grammar tests to pass so we can merge this in? Assigning back for you for now until the tests pass.

**Comment by zachlatta** [comment]
You'll want to rebase `master` into this pull request. I made a bunch of
spelling fixes in #453, which @MaxWofford merged last night.

On Thu, Jan 7, 2016 at 12:53 PM, Jonathan Leung <notifications@github.com>
wrote:

> @zachlatta <https://github.com/zachlatta>, what's the best way to get
> these spelling and grammar tests to pass so we can merge this in? Assigning
> back for you for now until the tests pass.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/meta/pull/451#issuecomment-169801968>.
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

**Comment by zachlatta** [comment]
Oh whoops, sorry (still early in the morning!). I didn't realize this was
my pull request.

I just rebased `master` into it and we should be good to go.

On Fri, Jan 8, 2016 at 8:21 AM, Zach Latta <zach@zachlatta.com> wrote:

> You'll want to rebase `master` into this pull request. I made a bunch of
> spelling fixes in #453, which @MaxWofford merged last night.
>
> On Thu, Jan 7, 2016 at 12:53 PM, Jonathan Leung <notifications@github.com>
> wrote:
>
>> @zachlatta <https://github.com/zachlatta>, what's the best way to get
>> these spelling and grammar tests to pass so we can merge this in? Assigning
>> back for you for now until the tests pass.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/hackclub/meta/pull/451#issuecomment-169801968>.
>>
>
>
>
> --
> Zach Latta
> http://zachlatta.com
> https://github.com/zachlatta
> https://twitter.com/zachlatta
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

**Comment by jonleung** [comment]
@zachlatta, feel free to assign me to this once you fixed the conflict!

**Comment by zachlatta** [comment]
Fixed!

File: club_leader_onboarding_protocol.md [hunk 1]
```diff
@@ -1,30 +1,58 @@
...
+   https://hackclub.io/apply. It's sent to the team as an email to
+   team@hackclub.io.
+2. If the application looks good, a team member responds to the applicant to set
+   up an interview call. They also add the application to the "Applications"
+   sheet in https://clubs.hackclub.io and update the application's status as the
+   application progresses (ex. the interview call gets scheduled).
+3. In the interview call the team member's goal is to get a good idea of the
+   applicant's leadership skills. Below are a few good questions to ask. The
+   team member should be taking notes. After the call is over, they should
+   upload their notes as a private gist and link them in the "Application" sheet
```
Comment (jonleung) [inline club_leader_onboarding_protocol.md]: I'm assuming you mean the "Applications" sheet:

> 
image

**Pull request merged**